src/components/views/rooms/ReplyTile.tsx

```diff
@@ -116,7 +117,7 @@ export default class ReplyTile extends React.PureComponent<IProps> {
         const msgtypeOverrides = {
             [MsgType.Image]: MImageReplyBody,
             // We don't want a download link for files, just the file name is enough.
-            [MsgType.File]: TextualBody,
+            [MsgType.File]: MFileBody,
             [MsgType.Audio]: TextualBody,
             [MsgType.Video]: TextualBody,
```

Comment:
Video & Audio show the same issue that File did which you just tweaked

![image](https://user-images.githubusercontent.com/2403652/125588979-d793187f-e06a-4c8c-812a-373926459aa0.png)

Reply:
Stickers even worse :(

![image](https://user-images.githubusercontent.com/2403652/125589091-d15d24ee-f703-4cc0-bff8-89453ed68f00.png)

Reply:
I think there might be a cheap and cheerful solution we can do here where we just put an icon to the left of the textual body in CSS, like a paperclip style icon for files/videos/etc and a sticker style icon for stickers to make it clearer that the content is summarised

Reply:
Hmm, true, right now I wouldn't mind using the file body for all. What do you think of that?

Reply:
Maybe, though would be nice if we could hide the download/decrypt via CSS and just have that pill at that point, running it by design real quick

Reply:
Yeah, sure

Reply:
![Screenshot_20210714_104739](https://user-images.githubusercontent.com/25768714/125592801-28282e10-d803-4db4-babe-feaf7df07b8e.png)

This is how it looks

Reply:
> I think there might be a cheap and cheerful solution we can do here where we just put an icon to the left of the textual body in CSS, like a paperclip style icon for files/videos/etc and a sticker style icon for stickers to make it clearer that the content is summarised

GH decided not to show me this message... Yeah, this sounds great!

Reply:
Nique gave a 👍 to the pill thing for files
For audio/video it looks like the path forward will be that pill but with an alternative icon instead of paperclip. Should be doable relatively sanely given the icon is set in CSS.
Designs incoming.

As for stickers, treatment similar to images where we show it much much smaller would be desired.

Reply:
Great!

> Should be doable relatively sanely given the icon is set in CSS.

I wonder if it's preferable to pass the icon as a prop or to override from `_ReplyTile.scss`?

Reply:
Passing it as a prop will get messy as you'd need to set it as a `--var` and keep it updated so that the CSS `::before` selector can access it, I'd say override from `_ReplyTile.scss` will be cleaner and simpler.

Reply:
![Screenshot_20210714_121212](https://user-images.githubusercontent.com/25768714/125605072-f1df6b36-febd-49af-b09e-8dee6c3b0cbf.png)

Sticker using `MImageReplyBody`

Reply:
That works way better than I expected

:shipit: